### PR TITLE
Example: password

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

**Comment by roflmyrlok** [comment]
#15

**Review by roflmyrlok — CHANGES_REQUESTED** [review]

**Comment by roflmyrlok** [comment]
```py

#Generate random password python translated
import random
import random

def generate_password(length_of_password):
  key = []
  for i in range(length_of_password):
    key.append(random.choice(characters))
  result = ''.join(key)
  return result

characters = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '!', '@', '#', '$', '%', '^', '&', '*']
result = generate_password(8)
print(result)

```

File: Tasks/password.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Generate random password
+
+let GeneratePassword = ( alphabet , length ) => {
```
Comment (roflmyrlok) [inline Tasks/password.js:3]:
```suggestion
const generatePassword = (alphabet, length) => {
```

**Comment by roflmyrlok** [comment]
```py
import random

def generate_password(length_of_password):
  key = []
  for i in range(length_of_password):
    key.append(random.choice(characters))
  result = ''.join(key)
  return result


characters = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '!', '@', '#', '$', '%', '^', '&', '*']
result = generate_password(8)
print(result)
```

**Comment by roflmyrlok** [comment]
```js
// Generate random password fixed

const generatePassword = (lengthOfPassword) => {
  let key = [];
  for (let i = 0; i < lengthOfPassword; i++) {
    key.push(characters[Math.floor(Math.random() * characters.length)]);
  }
  return key.join("");
};

const characters = [
  "0",
  "1",
  "2",
  "3",
  "4",
  "5",
  "6",
  "7",
  "8",
  "9",
  "a",
  "b",
  "c",
  "d",
  "e",
  "f",
  "g",
  "h",
  "i",
  "j",
  "k",
  "l",
  "m",
  "n",
  "o",
  "p",
  "q",
  "r",
  "s",
  "t",
  "u",
  "v",
  "w",
  "x",
  "y",
  "z",
  "A",
  "B",
  "C",
  "D",
  "E",
  "F",
  "G",
  "H",
  "I",
  "J",
  "K",
  "L",
  "M",
  "N",
  "O",
  "P",
  "Q",
  "R",
  "S",
  "T",
  "U",
  "V",
  "W",
  "X",
  "Y",
  "Z",
  "!",
  "@",
  "#",
  "$",
  "%",
  "^",
  "&",
  "*",
];
const result = generatePassword(8);
console.log(result);

```

**Comment by tshemsedinov** [comment]
- Please extract random to separate function
- Use const

**Comment by roflmyrlok** [comment]
```js
// Generate random password fixed
const random = (max) => Math.floor(Math.random() * max);

const generatePassword = (lengthOfPassword) => {
  const key = [];
  for (let i = 0; i < lengthOfPassword; i++) {
    const randomChar = characters[random(characters.length)]
    key.push(randomChar);
  }
  return key.join("");
};


    
const characters = [
  "0",
  "1",
  "2",
  "3",
  "4",
  "5",
  "6",
  "7",
  "8",
  "9",
  "a",
  "b",
  "c",
  "d",
  "e",
  "f",
  "g",
  "h",
  "i",
  "j",
  "k",
  "l",
  "m",
  "n",
  "o",
  "p",
  "q",
  "r",
  "s",
  "t",
  "u",
  "v",
  "w",
  "x",
  "y",
  "z",
  "A",
  "B",
  "C",
  "D",
  "E",
  "F",
  "G",
  "H",
  "I",
  "J",
  "K",
  "L",
  "M",
  "N",
  "O",
  "P",
  "Q",
  "R",
  "S",
  "T",
  "U",
  "V",
  "W",
  "X",
  "Y",
  "Z",
  "!",
  "@",
  "#",
  "$",
  "%",
  "^",
  "&",
  "*",
];
const result = generatePassword(8);
console.log(result);
```

**Pull request closed without merging**